src/App.tsx

```diff
@@ -42,16 +45,52 @@ const App: React.FC = (): ReactElement => {
 
   function parseAttemptHistoryFromFile(fileObject: any): Attempt[] {
     const history: Attempt[] = fileObject?.Run?.AttemptHistory?.[0]?.Attempt?.flatMap(
-      ({ $, RealTime }: any) => {
-        return $?.id && RealTime?.[0]
-          ? { x: Number($.id), y: moment.duration(RealTime[0]).as('milliseconds') }
+      ({ id, RealTime }: any) => {
+        return id && RealTime
+          ? { x: Number(id), y: moment.duration(RealTime).as('milliseconds') }
```

Comment:
Does this not need to be wrapped in an array? I thought it did for flatmap to pass it through, but I suppose an object can't be flattened anyway so perhaps it doesn't care and works anyway.

Reply:
Doesn't appear to need to be, however if it's more correct I'll make that change.